examples/tut-10/algoMAW.patch

```diff
@@ -0,0 +1,13 @@
+--- examples/tut-9/index.js	2021-08-31 12:43:24.801893800 -0400
```

Comment:
Have you tested with these patches? I assume you have not and should remove them